lfs/upload_queue.go

```diff
@@ -34,6 +34,20 @@ func NewUploadable(oid, filename string, index, totalFiles int) (*Uploadable, *W
 	return &Uploadable{oid: oid, OidPath: path, Filename: filename, size: fi.Size()}, nil
 }
 
+func NewUploadableWithoutFilename(oid string) (*Uploadable, *WrappedError) {
```

Comment:
ok ce20dd483602bcd21d8640e0ebace39c22d3ea30
although I am not particular happy with the variable name `pathForStats`. Ideas appreciated :smile: